html-block-and-inline/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>HTML testing</title>
+  </head>
+  <body>
+    <h1 style="font-weight: bolder; text-transform: uppercase">
+      Blue armerica snuggie&#xae; blanket
+    </h1>
+    <h2>By snuggie</h2>
+
+    <p>price: <s>$19.99</s> $14.99</p>
+
+    <p>
+      <span style="text-transform: capitalize">part</span> of our new
+      <span style="text-transform: capitalize">armericana</span> collection,
+      this <em>buetiful</em> blue is already a customer favorite
+    </p>
+
+    <p>Enjoy the <span style="font-weight: bold">Freedm of movement </span></p>
```

Comment:
Instead of giving it the styling of `bold`, you should be using the `strong` HTML element